src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FlexTicketDiscountInfo.tsx

```diff
@@ -35,7 +35,6 @@ export const FlexTicketDiscountInfo = ({userProfiles, style}: Props) => {
   const {appTexts} = useFirestoreConfiguration();
   const {flex_ticket_url} = useRemoteConfig();
 
-  if (!userProfiles.some((u) => u.offer.flex_discount_ladder)) return null;
```

Comment:
I guess this is intentional, but this seems to skip check either if a discount is applicable or have had discounts from before.

Reply:
Thank you, I missed this part! 😅

Reply:
It needs to be changed, but this shouldn't be in this commit, thanks for noticing it!